File: StudyGamePrograming06re/Plane.cpp
#include "Plane.h"
#include "MeshComponent.h"
#include "Game.h"
#include "Renderer.h"

Plane::Plane(Game* game) : Actor(game)
{
	MeshComponent* mc = new MeshComponent(this);
	mc->SetMesh(game->GetRenderer()->GetMesh("Assets/Plane.gpmesh"));
}

File: StudyGamePrograming06_VB/Plane.vb
Public Class Plane
    Inherits Actor
    'public
    Sub New(ByRef game As Game)
        MyBase.New(game)
        Dim mc As MeshComponent = New MeshComponent(Me)
        mc.SetMesh(game.GetRenderer.GetMesh("Assets/Plane.gpmesh"))
    End Sub
End Class
